src/OrchardCore.Modules/OrchardCore.ContentFields/Views/DateField.Edit.cshtml

```diff
@@ -6,11 +6,12 @@
 
 <script asp-name="jQuery-ui-i18n" at="Foot"></script>
 <div class="@Orchard.GetFieldWrapperClasses(Model.PartFieldDefinition)" id="@Html.IdFor(x => x.Value)_FieldWrapper">
```

Comment:
Have you tried with `asp-validation-for="Value"` on this div to set the entire "field" to be set has having a validation error?

Reply:
Example: 

![image](https://github.com/OrchardCMS/OrchardCore/assets/3228637/99d37d1b-4c3c-4f1d-be6e-73b4cedac4c7)